pkg/golinters/revive/revive.go

```diff
@@ -447,3 +452,36 @@ func defaultConfig() *lint.Config {
 	}
 	return &defaultConfig
 }
+
+func displayRules(conf *lint.Config) {
+	if !isDebug {
+		return
+	}
+
+	var enabledRules []string
+	for k, r := range conf.Rules {
+		if !r.Disabled {
+			enabledRules = append(enabledRules, k)
+		}
+	}
+
+	sort.Strings(enabledRules)
+
+	debugf("All available rules (%d): %s.", len(allRules), strings.Join(extractRulesName(allRules), ", "))
+	debugf("Default rules (%d): %s.", len(allRules), strings.Join(extractRulesName(allRules), ", "))
+	debugf("Enabled by config rules (%d): %s.", len(enabledRules), strings.Join(enabledRules, ", "))
+
+	debugf("revive configuration: %#v", conf)
+}
+
+func extractRulesName(rules []lint.Rule) []string {
+	var names []string
```

Comment:
Looks like we can pre-allocate:

```suggestion
	names := make([]string, 0, len(rules))
```

Reply:
no pre-declaration unless you have performance problem.